example/gardener-local/kube-apiserver/authz-config-with-seedauthorizer.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: apiserver.config.k8s.io/v1beta1
+kind: AuthorizationConfiguration
+authorizers:
+- type: RBAC
+  name: rbac
+- type: Node
+  name: node
+- type: Webhook
+  name: seed-authorizer
+  webhook:
+    authorizedTTL: 0s
+    unauthorizedTTL: 0s
```

```suggestion
    authorizedTTL: 1ns
    unauthorizedTTL: 1ns
```
Otherwise, the TTL will be `5m` and `30s` because of defaulting (see [here](https://github.com/kubernetes/apiserver/blob/3658357fea9fa8b36173d072f2d548f135049e05/pkg/apis/apiserver/v1/defaults.go#L52-L59))